snuba/web/query.py

```diff
@@ -174,7 +200,13 @@ def raw_query(
                         )
             except RateLimitExceeded as ex:
                 stats = log_query_and_update_stats(
-                    request, sql, timer, stats, "rate-limited", query_settings
+                    request,
+                    sql,
+                    timer,
+                    stats,
+                    "rate-limited",
+                    query_list,
+                    query_settings,
```

Comment:
might also pass the trace_id all the way through to here if that would be useful to people

Reply:
Having the trace_id would let us co-relate logs and APM data which would be nice.

Reply:
added